modules/ossm-auto-route.adoc

```diff
@@ -64,7 +64,7 @@ Then, the following OpenShift Routes are created automatically. You can check th
 
 [source,terminal]
 ----
-$ oc -n <your-control-plane-namespace> get routes
+$ oc -n <control_plane_namespace> get routes
```

Comment:
Commenting here about this IOR section. I think it could be rewritten as:

**Disabling Automatic Route Creation**

By default, Red Hat OpenShift Service Mesh control plane automatically synchronizes the Gateway resources with OpenShift routes.

If you do not wish this integration between Istio Gateways and OpenShift Routes to happen, you can disable it by setting the `ServiceMeshControlPlane` field `gateways.openshiftRoute.enabled` to false. For example, see the following resource snippet:

```yaml
spec:
  gateways:
    openshiftRoute:
      enabled: false
```

@neal-timpe Note that "Enabling" became "Disabling",  because it comes enabled by default in 2.0, in contrast with 1.1, that had to be manually enabled. Also, this sentence: "If the Gateway contains a TLS section, the OpenShift Route will be configured to support TLS." I think could be moved up to the section above (Automatic Route creation)